### DISCUSSION TITLE
How do we can get great circles in pole figure?

### DISCUSSION THREAD

**Discussion by SarveshUKY** [mtex-toolbox/mtex#2206] [Ask Anything]
**Hello all, 

When I plot the pole figure of copper using the program detailed below, is it possible to get the great circles overlapped in the pole figure?** 

Thank you for assistance. 

" h = {Miller(1,0,0,CS{1,2}),Miller(1,1,0,CS{1,2}),Miller(1,1,1,CS{1,2})};
figure()
plotPDF(odf_Cu,h,'antipodal','silent','colorrange','equal')
mtexColorbar
saveas(gcf,'PoleFigure_Cu','tiff'); "

**Comment by zmichels** [comment]
You can plot great circle traces by using the 'plane' flag in plot commands. I'm not sure this is what you are asking tho.

**Reply by SarveshUKY** [reply]
Yes, you are right, I am asking about the great circle traces in pole figure. Do you mean something like this ?plotPDF(odf_Cu, h, 'antipodal', 'plane', ....); If this is what you are suggesting?

**Comment by zmichels** [comment]
Sort of, yes. While I think can use ‘plane’ in the plotPDF() command, I believe you need to use orientations, rather than an ODF. Each great circle will correspond to a discrete direction-normal plane, so a *function* (ODF) won’t work… but one could sample from the ODF using discreteSample().At the bottom of the following link, there is a brief example of the syntax to use ‘plane’ in a plot() command with Millers:Miller Indices | MTEXmtex-toolbox.github.ioOn Sep 2, 2024, at 4:19 AM, SarveshUKY ***@***.***> wrote:﻿
Yes, you are right, I am asking about the great circle traces in pole figure. Do you mean something like this ?plotPDF(odf_Cu, h, 'antipodal', 'plane', ....); If this is what you are suggesting?
